data_management/opensearch_indexer/opensearch_indexer/text_extraction.py

```diff
@@ -76,5 +80,56 @@ def extract_text(file_stream: bytes, file_extension: str) -> str:
     ) as temp:
         temp.write(file_stream)
         temp.flush()
-        context = textract.process(temp.name)
-    return context.decode("utf-8")
+        file_path = temp.name
+
+        try:
+            context = textract.process(file_path)
+            return context.decode("utf-8")
+
+        except Exception as e:
+            logger.warning(f"Textract failed on {file_path}: {e}")
+
+            if file_extension not in CONVERT_MAP:
+                raise e
+            try:
+                logger.info("Attempting to convert via LibreOffice...")
```

Comment:
```suggestion
                logger.info("Attempting to convert to {output_file_type} before trying textract again...")
```

to describe why we are doing the conversioon